packages/progress-bar/src/ProgressBar/ProgressBar.tsx

```diff
@@ -95,10 +110,21 @@ export function ProgressBar(props: ProgressBarProps) {
       </div>
 
       {description && (
-        <Description darkMode={darkMode} disabled={disabled}>
+        <Description id={descId} darkMode={darkMode} disabled={disabled}>
           {description}
         </Description>
       )}
+
+      {screenReaderMessage && (
+        <div
+          role="status"
+          aria-live="polite"
+          aria-atomic="true"
+          className={getInvisibleStyles()}
```

Comment:
I believe there should be some aria connection between this live region and the main progress bar. `aria-controls`/`aria-controlledby` maybe?